### PR TITLE
add show page

### PR DESCRIPTION
**Pull request by philiptang42**

### CODE REVIEW

File: app/controllers/food_trucks_controller.rb [hunk 1]
```diff
@@ -2,4 +2,9 @@ class FoodTrucksController < ApplicationController
   def index
     @food_trucks = FoodTruck.all
   end
+
+  def show
+    @food_truck = FoodTruck.find(params[:id])
+    # @food_trucks = @food_truck.reviews
```
Comment (davidrf) [inline app/controllers/food_trucks_controller.rb]: Delete this line

File: spec/features/tests/index_view_spec.rb [hunk 2]
```diff
@@ -18,9 +18,6 @@
 
     visit "food_trucks#index"
```
Comment (davidrf) [inline spec/features/tests/index_view_spec.rb]: Use rails path helpers instead

File: spec/features/tests/show_view_spec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+feature "user views show page", %{
+  As a user
+  I want to view a food truck's information
+  so that I can learn more about it
+} do
+  # Acceptance Criteria
+  # * If a user goes to the show page, they see a list of food trucks.
+
+  scenario "user views show page of a food truck" do
+    a = FoodTruck.create(
```
Comment (davidrf) [inline spec/features/tests/show_view_spec.rb]: assign to a better variable name

File: spec/features/tests/show_view_spec.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      name: "Chicken & Rice Guys",
+      description: "Food truck that specializes in chicken and lamb shawarma.",
+      avg_rating: 4,
+      location: "Harvard Square"
+    )
+
+    visit "/"
+    expect(page).to have_content("Chicken & Rice Guys")
```
Comment (davidrf) [inline spec/features/tests/show_view_spec.rb]: use the created object `i.e. expect(page).to have_content(food_truck.name)`

File: spec/features/tests/show_view_spec.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    expect(page).to have_content("4")
+
+    click_link("#{a.name}")
+    expect(page).to have_content("Chicken & Rice Guys")
+    expect(page).to have_content("Food truck that specializes in chicken and
+     lamb shawarma.")
+    expect(page).to have_content("4")
+    expect(page).to have_content("Harvard Square")
```
Comment (davidrf) [inline spec/features/tests/show_view_spec.rb]: Same as above for all these

File: spec/features/tests/show_view_spec.rb [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+  scenario "user views show page of a food truck" do
+    a = FoodTruck.create(
+      name: "Chicken & Rice Guys",
+      description: "Food truck that specializes in chicken and lamb shawarma.",
+      avg_rating: 4,
+      location: "Harvard Square"
+    )
+
+    visit "/"
```
Comment (davidrf) [inline spec/features/tests/show_view_spec.rb]: use rails path helpers

File: spec/features/tests/index_view_spec.rb [hunk 7]
```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
```
Comment (mlg-) [inline spec/features/tests/index_view_spec.rb]: The name of this spec file should be more descriptive. Can you call it user_views_all_food_trucks_spec.rb or something along those lines? The idea is that scanning through 50 files, you could tell which one pertains to whatever functionality you want to learn about.

File: spec/features/tests/show_view_spec.rb [hunk 8]
```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
```
Comment (mlg-) [inline spec/features/tests/show_view_spec.rb]: Same principle with the name of this test.

File: spec/features/tests/show_view_spec.rb [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+    )
+
+    visit food_truck_path(sample_food_truck_1.id)
+    expect(page).to have_content(sample_food_truck_1.name)
+    expect(page).to have_content(sample_food_truck_1.description)
+    expect(page).to have_content(sample_food_truck_1.avg_rating)
+    expect(page).to have_content(sample_food_truck_1.location)
+
+  end
+end
```
Comment (mlg-) [inline spec/features/tests/show_view_spec.rb]: It looks to me like you built in a link to a given show page from the index. A test that this link works would be appropriate here, in addition to the one above that navigates to the path directly.

File: spec/features/tests/show_view_spec.rb [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+
+  scenario "user views show page of a food truck" do
+    sample_food_truck_1 = FoodTruck.create(
+      name: "Chicken & Rice Guys",
+      description: "Food truck that specializes in chicken and lamb shawarma.",
+      avg_rating: 4,
+      location: "Harvard Square"
+    )
+
+    visit food_truck_path(sample_food_truck_1.id)
```
Comment (mlg-) [inline spec/features/tests/show_view_spec.rb]: You should be able to access the path just by passing in the object; no need for `.id`.

**Pull request merged**